encryption/src/main/java/com/diyasylum/encryption/SecretBox.java

```diff
@@ -30,6 +30,7 @@ public SecretBox(String password) {
 
   public Optional<String> decrypt(EncryptedMessage encryptedMessage) {
     try {
+      // TODO: Add input validation
```

Comment:
I suggest filing this as an issue with a description and having this comment be a link to that